wallet/node/Main.hs

```diff
@@ -0,0 +1,188 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
```

Comment:
How is this different from the other wallets? Why is this in the `wallet` package rather than `wallet-new` ? It wa smy understanding that the `wallet` tree was going to be replaced with `wallet-new`.

Reply:
Maybe this was a file the was deleted on develop and reintroduced by mistake while rebasing.